skimage/draw/draw.py

```diff
@@ -41,35 +41,35 @@ def _ellipse_in_shape(shape, center, radii, rotation=0.):
     return np.nonzero(distances < 1)
 
 
-def ellipse(r, c, r_radius, c_radius, shape=None, rotation=0.):
-    """Generate coordinates of pixels within ellipse.
+def ellipse(r, c, r_radius, c_radius, orientation=0, shape=None, rotation=None):
+    """Generate coordinates of pixels corresponding to an ellipse.
 
     Parameters
     ----------
-    r, c : double
-        Centre coordinate of ellipse.
-    r_radius, c_radius : double
-        Minor and major semi-axes. ``(r/r_radius)**2 + (c/c_radius)**2 = 1``.
-    shape : tuple, optional
+    r, c : float
+        Center coordinate of the ellipse.
+    r_radius, c_radius : float
+        Minor and major semi-axes, where ``(r/r_radius)**2 + (c/c_radius)**2 = 1``.
+    orientation : float, optional (default : 0)
+        Major axis orientation in clockwise direction as radians.
+    shape : tuple, optional (default : None)
         Image shape which is used to determine the maximum extent of output pixel
         coordinates. This is useful for ellipses which exceed the image size.
-        By default the full extent of the ellipse are used.
-    rotation : float, optional (default 0.)
-        Set the ellipse rotation (rotation) in range (-PI, PI)
-        in contra clock wise direction, so PI/2 degree means swap ellipse axis
```

Comment:
So you changed it from counter clockwise to clockwise?

Reply:
Didn't change anything; only the variable names. `rotation` will become `orientation`, and `shape` and `orientation` will have their positions changed; this way it'll be equal to `ellipse_perimeter`, and we can receive input from, say, `EllipseModel()` and pass it directly to `ellipse` as `*parameters`.

Reply:
Old
> contra clock wise direction

New
> clockwise direction

Reply:
My bad, I'll correct it.
However, it makes no sense to have two similar functions behaving differently. Maybe it's time to get this equal to `ellipse_perimeter`

Reply:
My bad, I'll correct it.
However, it makes no sense to have two similar functions behaving differently. Maybe it's time to get this equal to `ellipse_perimeter`